website/versioned_docs/version-V1/sdk-reference/unity.mdx

```diff
@@ -0,0 +1,191 @@
+
+```cs
+using System;
+using System.Globalization;
+using System.Threading;
+using ConfigCat.Client;
+using UnityEngine;
+
+public class SingletonServices : MonoBehaviour
+{
+    private static SingletonServices instance;
+    public static SingletonServices Instance => instance;
+
+    [field: NonSerialized]
+    public IConfigCatClient ConfigCatClient { get; private set; }
+
+    private void Awake()
+    {
+        if (Interlocked.CompareExchange(ref instance, this, null) is not null)
+        {
+            // If another instance has been created already, get rid of this one.
+            Destroy(this);
+            return;
+        }
+
+        ConfigCatClient = ConfigCat.Client.ConfigCatClient.Get("YOUR_SDK_KEY", options =>
+        {
+            options.Logger = new ConfigCatToUnityDebugLogAdapter(LogLevel.Warning);
+        });
+
+        DontDestroyOnLoad(this.gameObject);
+    }
+
+    // Start is called before the first frame update
+    void Start() { }
+
+    // Update is called once per frame
+    void Update() { }
+
+    private void OnDestroy()
+    {
+        this.ConfigCatClient?.Dispose();
+        this.ConfigCatClient = null;
+    }
+
+    private sealed class ConfigCatToUnityDebugLogAdapter : IConfigCatLogger
+    {
+        private readonly LogLevel logLevel;
+
+        public ConfigCatToUnityDebugLogAdapter(LogLevel logLevel)
+        {
+            this.logLevel = logLevel;
+        }
+
+        public LogLevel LogLevel
+        {
+            get => this.logLevel;
+            set { throw new NotSupportedException(); }
+        }
+
+        public void Log(LogLevel level, LogEventId eventId, ref FormattableLogMessage message, Exception exception = null)
+        {
+            var eventIdString = eventId.Id.ToString(CultureInfo.InvariantCulture);
+            var exceptionString = exception is null ? string.Empty : Environment.NewLine + exception;
+            var logMessage = $"ConfigCat[{eventIdString}] {message.InvariantFormattedMessage}{exceptionString}";
+            switch (level)
+            {
+                case LogLevel.Error:
+                    Debug.LogError(logMessage);
+                    break;
+                case LogLevel.Warning:
+                    Debug.LogWarning(logMessage);
+                    break;
+                case LogLevel.Info:
+                    Debug.Log(logMessage);
+                    break;
+                case LogLevel.Debug:
+                    Debug.Log(logMessage);
+                    break;
+            }
+        }
+    }
+}
+```
+
+:::caution
+In case your application or game targets WebGL, this won't work. WebGL is not supported currently.
+:::
+
+Finally, to activate the `SingletonServices` script, add an empty GameObject to your first scene and assign the script to it.
+
+### 4. Get your setting value
+
+Once the setup above is done, you will be able to access the _ConfigCat_ client in your other scripts and use it to evaluate feature flags and settings:
```

Comment:
Itt is megkérdezem, hogy a "ConfigCat client" résznél biztosan szándékos-e ez a fajta dőltbetűsítés?

Reply:
L. feljebb.